python/api_post.py

```
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+import click
+import requests
+
+api_key_file = Path('/tmp/supersecret.txt')
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/supersecret.txt"

(S108)